lib/nr-security-agent/lib/core/commonUtils.js

```diff
@@ -384,7 +384,7 @@ function getCSECmode(){
  * @returns 
  */
 function getPathOfCACert(){
-    const certPath = NRAgent && NRAgent.config.security.ca_bundle_path ? NRAgent.config.security.ca_bundle_path : path.resolve(__dirname, '../../resources/cert.pem');
+    const certPath = process.env.NEW_RELIC_SECURITY_CA_BUNDLE_PATH ? process.env.NEW_RELIC_SECURITY_CA_BUNDLE_PATH : path.resolve(__dirname, '../../resources/cert.pem');
```

Comment:
by referencing the env vars you have cases where cusotmers are setting this in configuration. in this case it wouldn't be read

Reply:
got it, I will revert this change and this part will be added in documentation to provide custom certificate.